packages/gatsby-theme/src/gatsby-plugin-theme-ui/index.js

```diff
@@ -7,6 +7,7 @@ const bColors = {
   secondary: "#CA3E73",
   darkShade: "#37375B",
   lightShade: "#F7F7F7",
+  darkBlueOpaque: "rgba(21, 30, 46, 0.85)",
```

Comment:
This is up to you but we can change this to it's `hex` value `#151E2E` just to have more consistent values.

Reply:
I think rgba value is needed to have transparency effect. what do you think?

Reply:
Leave it as it is :smile: